### PR TITLE
fix(gaussdb): fix gaussdb opengauss/redis resource lint error

### PR DESCRIPTION
**Pull request by wangyuancheng01**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. region = "cn-north-4".
2. the TestAccPreCheckHighCostAllow method in gaussdb_opengauss_instance not allow expensive testing.
3. run redis instance with volume_size = 16.

```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/gaussdb" TESTARGS="-run TestAccOpenGaussInstance_basic"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb -v -run TestAccOpenGaussInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccOpenGaussInstance_basic
=== PAUSE TestAccOpenGaussInstance_basic
=== CONT  TestAccOpenGaussInstance_basic
    acceptance.go:640: Do not allow expensive testing
--- SKIP: TestAccOpenGaussInstance_basic (0.00s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   0.043s
```

```
make testacc TEST="./huaweicloud/services/acceptance/gaussdb" TESTARGS="-run TestAccGaussRedisEipAssociate_basic"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb -v -run TestAccGaussRedisEipAssociate_basic -timeout 360m -parallel 4
=== RUN   TestAccGaussRedisEipAssociate_basic
=== PAUSE TestAccGaussRedisEipAssociate_basic
=== CONT  TestAccGaussRedisEipAssociate_basic
--- PASS: TestAccGaussRedisEipAssociate_basic (1411.95s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   1411.998s
```

```
make testacc TEST="./huaweicloud/services/acceptance/gaussdb" TESTARGS="-run TestAccGaussRedisInstance_basic"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb -v -run TestAccGaussRedisInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccGaussRedisInstance_basic
=== PAUSE TestAccGaussRedisInstance_basic
=== CONT  TestAccGaussRedisInstance_basic
--- PASS: TestAccGaussRedisInstance_basic (3890.88s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   3890.937s
```

### CODE REVIEW

File: huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_opengauss_instance.go [hunk 1]
```diff
@@ -626,12 +625,11 @@ func resourceOpenGaussInstanceRead(_ context.Context, d *schema.ResourceData, me
 		return common.CheckDeletedDiag(d, err, "OpenGauss instance")
 	}
 	if instance.Id == "" {
-		d.SetId("")
```
Comment (ShiChangkuo) [inline huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_opengauss_instance.go]: please keep it as is

File: huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go [hunk 2]
```diff
@@ -467,23 +467,26 @@ func resourceGaussRedisInstanceV3Read(_ context.Context, d *schema.ResourceData,
 		return common.CheckDeletedDiag(d, err, "GaussRedis")
 	}
 	if instance.Id == "" {
-		d.SetId("")
```
Comment (ShiChangkuo) [inline huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go:470]: ditto

File: huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_opengauss_instance.go (hunk 1 above)
Comment (wangyuancheng01) [reply huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_opengauss_instance.go]: done

File: huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go (hunk 2 above)
Comment (wangyuancheng01) [reply huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go:470]: done

File: huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go [hunk 3]
```diff
@@ -527,7 +531,7 @@ func resourceGaussRedisInstanceV3Read(_ context.Context, d *schema.ResourceData,
 			d.Set("volume_size", volSize)
 		}
 		if specCode != "" {
-			logp.Printf("[DEBUG] Node SpecCode: %s", specCode)
+			log.Printf("[DEBUG] node specCode: %s", specCode)
 			d.Set("flavor", specCode)
```
Comment (ShiChangkuo) [inline huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go]: we should also use multierror.Append to get the errors.

Comment (wangyuancheng01) [reply huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go]: done

**Pull request merged**